cmd/flags_quality.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/mysteriumnetwork/node/core/node"
+	"github.com/urfave/cli"
+)
+
+var (
+	qualityTypeFlag = cli.StringFlag{
+		Name: "quality.type",
+		Usage: fmt.Sprintf(
+			"Quality Oracle provider (%s, %s - %s)",
```

Comment:
Let's try to keep the same option enumeration format as in e.g. `services/openvpn/service/options.go:36`